### PR TITLE
add diaproteomics biorxiv preprint

### PR DESCRIPTION
**Pull request by Leon-Bichmann**

### CODE REVIEW

File: markdown/publications.md [hunk 1]
```diff
@@ -96,3 +96,13 @@ Article in _Nature_ describing scientific workflows (including nf-core):
...
 > doi: [10.1016/j.jgg.2018.06.005](https://doi.org/10.1016/j.jgg.2018.06.005)
+
+### [nf-core/diaproteomics](https://nf-co.re/diaproteomics)
+
+<!-- pub-stats 10.1101/2020.12.08.415844v1 -->
+> **DIAproteomics: A multi-functional data analysis pipeline for data-independent-acquisition proteomics and peptidomics**
+>
+> Leon Bichmann, Shubham Gupta, George Rosenberger, Leon Kuchenbecker, Timo Sachsenberg, Oliver Alka, Julianus Pfeuffer, Oliver Kohlbacher & Hannes Rost.
+>
+> [_BiorXiV_, 2020.12.08.415844 (2020)](https://www.biorxiv.org/content/10.1101/2020.12.08.415844v1)
```
Comment (ewels) [inline markdown/publications.md:107]:
```suggestion
> [_BiorXiV_, 2020.12.08.415844 (2020)](https://www.biorxiv.org/content/10.1101/2020.12.08.415844v1); doi: [10.1101/2020.12.08.415844](https://doi.org/10.1101/2020.12.08.415844)
```

**Review by ewels — APPROVED** [review]

**Comment by apeltzer** [comment]
Hi @Leon-Bichmann !

see also in slack in #website - would you be comfortable to help out here and make the list of publications alphabetical based on pipeline name used in nf-core? We're now seeing many publications and want to give this a bit more structure...

So please:


- sort already existing + your entry alphabetically
- add a ToC on top

This applies to only the publications that are not nf-core itself / so the list of publications :-)

If you're not having the time for this - let me know and I'll merge yours and do it myself, just thought it might be easiest to ask you

**Comment by Leon-Bichmann** [comment]
ok sure I can rearrange it in this PR

**Comment by Leon-Bichmann** [comment]
forgot to pull and force-pushed it now, is it ok like this?

**Comment by apeltzer** [comment]
Missing the Table of Contents for that one - but I can add this afterwards, otherwise looks good to me :-)

**Comment by Leon-Bichmann** [comment]
ok thanks then I merge it for now!

**Pull request merged**